src/sparql/orcids.sparql

```diff
@@ -0,0 +1,7 @@
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+prefix owl: <http://www.w3.org/2002/07/owl#>
+SELECT DISTINCT ?orcid
+WHERE {
+  ?term <http://purl.org/dc/terms/contributor> ?orcid . 
+  FILTER(isIRI(?term))
```

Comment:
This constraint could miss oio:hasDbXrefs on defs, FYI

Reply:
I might be redundant.